tests/lib/rules/no-whitespace-before-property.js

```diff
@@ -507,6 +510,46 @@ ruleTester.run("no-whitespace-before-property", rule, {
             code: "foo ['bar ' + 1 + ' baz']",
             output: "foo['bar ' + 1 + ' baz']",
             errors: ["Unexpected whitespace before property 'bar ' + 1 + ' baz'."]
+        },
+        {
+            code: "5       .toExponential()",
+            output: "5 .toExponential()",
```

Comment:
I'm not sure if spacing rules should fix codes except spaces.
Also, we don't have `number-literal-syntax`-like rule, so I'd like to leave the warning of `5 .toExponential()` in order to fix it manually for now. 

`eslint --fix` command does not report how/where the code was fixed. I think bad if it fixes codes in an unexpected way without notice. On the other hand, remaining problems are reported. People can fix those manually.